README.md

```diff
@@ -449,21 +460,22 @@ If your project has a Conan configuration error, you can use `conan info` to
 find it. `conan info` displays information about the dependency graph of your
 project, with colorized output in some terminals.
 
-    $ cd build
-    $ conan info .
+    cd build
+    conan info .
 
 In my terminal, the first couple lines of `conan info`'s output show all of the
 project's configuration warnings in a bright yellow font.
 
 For example, the package `spdlog/1.5.0` depends on the package `fmt/6.1.2`.
-If you were to modify the file `cmake/Conan.cmake` so that it requires an
+If you were to modify the file `conanfile.py` so that it requires an
 earlier version of `fmt`, such as `fmt/6.0.0`, and then run:
 
-    $ conan remove -f '*'       # clear Conan cache
-    $ rm -rf build              # clear previous CMake build
-    $ mkdir build && cd build
-    $ cmake ..                  # rebuild Conan dependencies
-    $ conan info .
+```bash
+conan remove -f '*'       # clear Conan cache
+rm -rf build              # clear previous CMake build
+cmake -S . -B ./build     # rebuild Conan dependencies
+conan info .
```

Comment:
This command is now broken.

```suggestion
conan info ./build
```

`conan info .` works properly in the build directory, but in the source directory, it often cannot determine some of the variables it needs. All the info it needs exists in the build directory.